packages/config/config/devices/0x001d/dz6hd.json

```diff
@@ -21,15 +21,27 @@
 			"minValue": 0,
 			"maxValue": 253,
 			"defaultValue": 2,
-			"unsigned": true
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Instant on",
+					"value": 0
+				}
+			]
 		},
 		"2": {
 			"label": "Fade Off Time",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 253,
 			"defaultValue": 2,
-			"unsigned": true
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Instant off",
+					"value": 0
+				}
+			]
 		},
 		"3": {
 			"label": "Minimum Light Level",
```

Comment:
Z-Wave levels usually go from 0 to 99.
Can you confirm that these actually go from 0 to 100 - so an actual percentage?
In this case we should add `"unit": "%"` to params 3 .. 5